<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab tab_name="tab1" containers="1">
   <Container>
    <DockSplitter orientation="-" count="1" sizes="1">
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range top="4000.550000" left="4.231894" bottom="1037.450000" right="24.289609"/>
       <limitY/>
       <curve color="#1f77b4" name="/uwb/range_unpacked/592B_59AE/range_rsl"/>
       <curve color="#d62728" name="/uwb/range_unpacked/592B_5993/range_rsl"/>
       <curve color="#1ac938" name="/uwb/range_unpacked/5826_59AE/range_rsl"/>
       <curve color="#ff7f0e" name="/uwb/range_unpacked/5826_592B/range_rsl"/>
       <curve color="#f14cc1" name="/uwb/range_unpacked/5826_5993/range_rsl"/>
       <curve color="#9467bd" name="/uwb/range_unpacked/5993_59AE/range_rsl"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin status="idle" ID="CSV Exporter"/>
  <plugin status="idle" ID="ROS /rosout Visualization"/>
  <plugin status="idle" ID="ROS Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

